recipes/fsc.export/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "fsc.export" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/fsc.export-{{ version }}.tar.gz
+  sha256: 3886d5990b7b482cc889ae8d9e29c5062b5c1edaa9cb4f6aa3e67f0e4d629535
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - fsc.export
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: http://frescolinogroup.github.io/frescolino/pyexport/1.2
+  summary: Decorator to add objects to their module's __all__.
+  license: Apache-1.1
```

Comment:
Could you clarify where it says that this is Apache 1.1 licensed? I only see a generic "apache license" in the source code.

Reply:
Ah this was just auto-added by https://github.com/conda-incubator/grayskull, can change it to just `Apache` if you want?

Reply:
changes to apache-2.0

Reply:
Sorry, but do we know whether this is apache 2.0?

Reply:
Well I guess we don't exactly; but what does a non-versioned Apache license default to? Or are you saying that it cannot be accepted until the license is changed at source?

Reply:
It's my understanding that these licenses are quite different. Can we check with upstream what the license actually is?

If upstream does not respond, we can discuss what to make of this but then we probably need to include more people (I am not sure what's the conda-forge policy here.)

Reply:
Hey @saraedum, I checked what the license is here: https://github.com/greschd/aiida-tools/issues/3; it was apache 2.0, and I have updated it as suggested there

Reply:
is that ok?